README.md

```diff
@@ -1 +1,8 @@
 Light & Motion project. Purpose: To read in a CSV data file and output its raw data and graphic visualization into an Excel file. Currently only works with an Lumensphere CSV file. 
+
+* Derived Data Imjin 800.csv- raw CSV file 
```

Comment:
Trying to figure out how reviewing works.

Reply:
Seems like a good opportunity for us to mess around with the features, this is great!